doc/doxygen/headers/manifold.h

```diff
@@ -202,27 +205,20 @@
  *               outer_radius = 1.0;
  *  GridGenerator::hyper_shell (triangulation,
  *                              center, inner_radius, outer_radius,
- *                              4);    // four circumferential cells
+ *                              3);    // four circumferential cells
```

Comment:
Haven't looked at the rest of the patch, but this at least is funny ;-)

Reply:
So much about comments and code that should be in phase... 😃